frontend/src/component/admin/network/NetworkTrafficUsage/NetworkTrafficUsage.tsx

```diff
@@ -192,7 +209,11 @@ const NewNetworkTrafficUsage: FC = () => {
                         },
                     );
                 } else {
-                    return new Date(tooltipItems[0].label).toLocaleDateString(
+                    const timestamp = Date.parse(tooltipItems[0].label);
+                    if (Number.isNaN(timestamp)) {
```

Comment:
Interesting! so essentially .. Date.parse('current') -> NaN ?

Reply:
Yup. [According to mdn](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/parse), it returns:
> A number representing the [timestamp](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date#the_epoch_timestamps_and_invalid_date) of the given date. If dateString fails to be parsed as a valid date, [NaN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/NaN) is returned.

Reply:
Of course, we could also check whether the label is equal to "Current", but that'd break if the we changed the label, so I went with this strange check.